packages/common/src/hooks/useEffectAfterMounting/useEffectAfterMounting.test.ts

```diff
@@ -0,0 +1,48 @@
+import { useEffectAfterMounting } from './useEffectAfterMounting';
```

Comment:
There's two hooks here `useEffectAfterMounting` and `useMountedRef`.

They were from previous iterations (Was triggering the callback when `value` changed, but not on the first render) until I just added the callback to `onBlur` which was a bit easier.

I left these in here as they work and might get use 🤷